src/main/java/pro/sky/java/school/ServiceImpl/ExaminerServiceImpl.java

```diff
@@ -0,0 +1,39 @@
+package pro.sky.java.school.ServiceImpl;
+
+import org.springframework.stereotype.Service;
+import pro.sky.java.school.Exception.LargeNumberOfQuestions;
+import pro.sky.java.school.Service.ExaminerService;
+import pro.sky.java.school.Service.QuestionService;
+import pro.sky.java.school.data.Question;
+
+import java.util.*;
+
+@Service
+//@SessionScope
+public class ExaminerServiceImpl implements ExaminerService {
+
+    public QuestionService questionService;
+
+    public ExaminerServiceImpl(QuestionService questionService) {
+        this.questionService = questionService;
+    }
+
+    @Override
+    public List<Question> getQuestions(int amount) {
+        Collection<Question> a = questionService.getAll();
```

Comment:
старайтесь избегать плохого именования переменных
```suggestion
        Collection<Question> allQuestions = questionService.getAll();
```